tsec-libsodium/src/main/scala/tsec/libsodium/ScalaSodium.scala

```diff
@@ -57,20 +57,24 @@ object ScalaSodium
     sodium
   }
 
-  def getSodiumUnsafe: ScalaSodium = Sodium
+  final def getSodiumUnsafe: ScalaSodium = Sodium
 
-  def getSodium[F[_]](implicit F: Sync[F]): F[ScalaSodium] = F.delay(Sodium)
+  final def getSodium[F[_]](implicit F: Sync[F]): F[ScalaSodium] = F.delay(Sodium)
 
-  def randomBytes[F[_]](len: Int)(implicit F: Sync[F], S: ScalaSodium): F[Array[Byte]] = F.delay {
+  final def randomBytes[F[_]](len: Int)(implicit F: Sync[F], S: ScalaSodium): F[Array[Byte]] = F.delay {
     val bytes = new Array[Byte](len)
     S.randombytes_buf(bytes, len)
     bytes
   }
 
-  def randomBytesUnsafe(len: Int)(implicit S: ScalaSodium): Array[Byte] = {
+  final def randomBytesUnsafe(len: Int)(implicit S: ScalaSodium): Array[Byte] = {
```

Comment:
_technically_ this one shouldn't make a difference b/c it's on an `object`, but cool